ember/README.md

```diff
@@ -0,0 +1,57 @@
+# caluma-portal-demo
+
+This README outlines the details of collaborating on this Ember application.
+A short introduction of this app could easily go here.
```

Comment:
Please write something meaningful here. What's this project about? 1 to 2 sentences should be enough. We are in a open source context, so people should know what this is about.

Reply:
Or delete it since it's just in the ember subfolder. :+1: